### ISSUE TITLE
Docker file for v7 agent ?

### ISSUE THREAD

**Issue by console-writeline** [JAMSScheduler/docker#1]
Hello JAMS support team - any chance of upgrading this S3 path for v7 jams agent now that you have upgraded from v6.5 to v7 ?

**Comment by JVottero** [comment]
Yes, we will be publishing Docker containers for JAMS V7. We will reply to this thread when they are available.
